util/src/test/java/tech/pegasys/artemis/util/mikuli/G2PointTest.java

```diff
@@ -356,7 +366,8 @@ void succeedsWhenAttemptToDeserialiseWithBFlagAndAFlagTrueThrowsIllegalArgumentE
    * The tests have since changed, and are now automated, but these remain here for reference.
    */
 
-  @Test
+  @Disabled
+  // TODO: update vectors for SHA256
```

Comment:
I'm not entirely sure how this is related to the hash code change.  Was it accidentally included?

Reply:
I'd understood that these had been disabled previously, but I found they weren't, so just making the update in passing. Can do a separate PR if required...